EnglishTranslation-master/ETC.tsv

```diff
@@ -23590,17 +23590,17 @@ ETC_20160811_023584	For the war against demons	마족과의 전쟁을 위해서
 ETC_20160811_023585	To show the might of the Royal Family	왕실의 권위를 나타내려고				
 ETC_20160811_023586	Medzio Diena has been stopped	신수의 날을 완벽하게 막아냈다				
 ETC_20160811_023587	It does not have any relation to Medzio Diena	신수의 날과는 관계가 없다				
-ETC_20160811_023588	It activated as planned but was not strong enough to stop Medzio Diena	제대로 작동했지만 신수의 힘이 너무 강력해서 막지 못했다				
+ETC_20160811_023588	Goddess Laima was not strong enough to stop Medzio Diena	제대로 작동했지만 신수의 힘이 너무 강력해서 막지 못했다				
 ETC_20160811_023589	Goddess Laima is actually a demon	라이마 여신은 사실 마족이다				
-ETC_20160811_023590	The Three-Layered Castle Wall is to actually amplify the power of the Divine Tree	사실 삼중성벽은 신수의 힘을 증폭시키기 위함이었다				
-ETC_20160811_023591	The fact that you feel evil energy here has nothing to do with it	마기가 느껴지는 것은 아무 상관 없다				
+ETC_20160811_023590	The Castle Walls were made to amplify the power of the Divine Tree	사실 삼중성벽은 신수의 힘을 증폭시키기 위함이었다				
```

Comment:
why removing the note about three layers?

Reply:
The dialogue window became extremely cut-off.

Anything more than these causes several words to be 'cut off' from the dialogue window entirely. In its own case, "The Tree-Layered castle" was unreadable because of the line length. We have to respect the character limitations for these windows, unfortunately.


These are the trimmed down answer versions:

![screen shot 12-05-17 at 03 22 am](https://user-images.githubusercontent.com/18273533/33588719-397798f0-d96c-11e7-95a7-0717a70425ad.PNG)

![screen shot 12-05-17 at 03 21 am](https://user-images.githubusercontent.com/18273533/33588678-fe98a170-d96b-11e7-8dbc-8736697841ca.PNG)